### PR TITLE
5088 - Fix Alignment on Lookup Close Button

### LINKED ISSUE #5088: LookUp: x icon in was misaligned in uplift
**Describe the bug**

**To Reproduce**

1. Go to https://main-enterprise.demo.design.infor.com/components/lookup/example-clearable.html
2. Input some text
3. See error

**Expected behavior**
x icon should be aligned

**Version**

- ids-enterprise: 4.50-main

**Screenshots**
image

**Platform**
Allmajor browsers

**Additional context**
see also
https://main-enterprise.demo.design.infor.com/components/lookup/example-filter-lookup.html
image

https://main-enterprise.demo.design.infor.com/components/lookup/example-icon.html
image

https://main-enterprise.demo.design.infor.com/components/lookup/example-index.html
image

https://main-enterprise.demo.design.infor.com/components/lookup/example-minwidth.html

### PR DESCRIPTION
**Pull request by rob2d**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Close/"X" button alignment on searchfield in lookup was misaligned in the new theme variant.

**Related github/jira issue (required)**:

Closes #5088.

**Steps necessary to review your pull request (required)**:

- pull `5088-lookup-x-icon-misalignment`
- visit `http://localhost:4000/components/lookup/example-clearable.html`
- type anything in one of the search fields and see that the X icon is not misaligned vertically
image
- repeat for `http://localhost:4000/components/lookup/example-clearable.html?theme=classic`

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [X] A note to the change log.

### CODE REVIEW

**Review by tmcconechy — COMMENTED** [review]
A bit picky but i think its 1 or two pixels too high. It looks ok in classic mode.

Screen Shot 2021-04-20 at 11 39 19 AM

Also a question for @EdwardCoyle and could be considered a new issue but should the x be tabbable it skips right be it when you tab in the field to the lookup.

**Review by tmcconechy — DISMISSED** [review]

**Review by ericangeles — CHANGES_REQUESTED** [review]
I saw an issue in RTL

- Go to http://localhost:4000/components/lookup/example-clearable.html?theme=new&mode=light&colors=2578a9&locale=ar-EG
- Open the lookup
- Search something to trigger the clearable 
- Clearable is overlapping the search icon

clearable

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -75,6 +75,7 @@
 - `[Input]` Changed the disabled search field color for Safari to match that of other browsers. ([#4611](https://github.com/infor-design/enterprise/issues/4611))
 - `[Lookup]` Isolated the scss/css .close.icon class inside of .modal-content and removed any extra top property to fix the alignment issue.([#4933](https://github.com/infor-design/enterprise/issues/4933))
 - `[Lookup]` Added a setting that replaces the trigger icon with an actual button for better accessibility, enabled by default. ([#4820](https://github.com/infor-design/enterprise/issues/4820))
+- `[Lookup]` fix close button alignment issue. ([#5088](https://github.com/infor-design/enterprise/issues/5088))
```
Comment (ericangeles) [inline docs/CHANGELOG.md:78]: A better changelog perhaps?

**Comment by rob2d** [comment]
@ericangeles good catch/testing, thanks. Also good point on the changelog -- slightly increased scope but better to just avoid revisiting these buttons 🙂 

(pushed fix)
rtl-modal-searchfield

File: docs/CHANGELOG.md (hunk 1 above)
Comment (rob2d) [reply docs/CHANGELOG.md:78]: think it should be good now -- good point/catch.

**Review by tmcconechy — APPROVED** [review]
This is good for my checks now - thanks @rob2d 👍🏻

**Comment by ericangeles** [comment]
Looks pretty good now! But another thing that I saw on classic mode

It's not perfectly centered when it is focused.

clearable-classic

**Comment by rob2d** [comment]
@ericangeles appreciate the thoroughness. Just went through/caught another case on LTR and retested all of them after these tweaks. Hopefully all in lookup close button cases covered (🤞).

**Review by tmcconechy — APPROVED** [review]

**Review by ericangeles — APPROVED** [review]
Sweet! 🍌 Thanks for the fix, @rob2d.

**Pull request merged**